Containers/nextcloud/Dockerfile

```diff
@@ -1,29 +1,34 @@
 # From https://github.com/nextcloud/docker/blob/master/23/fpm-alpine/Dockerfile
 FROM php:8.0.28-fpm-alpine3.16
 
+ENV PHP_MEMORY_LIMIT 512M
+ENV PHP_UPLOAD_LIMIT 10G
+ENV PHP_MAX_TIME 3600
+ENV NEXTCLOUD_VERSION 25.0.6
+
+COPY --chmod=775 *.sh /
+COPY upgrade.exclude /upgrade.exclude
+COPY config/* /usr/src/nextcloud/config/
```

Comment:
I fear this will only work after nextcloud was extracted into /usr/src ?

Reply:
possible